crates/nvim-api/src/buffer.rs

```diff
@@ -314,6 +309,8 @@ impl Buffer {
                 start,
                 end,
                 strict_indexing,
+                #[cfg(feature = "neovim-nightly")]
+                std::ptr::null_mut(),
```

Comment:
We can just use the initialized Lua state:

```rust
#[cfg(feature = "neovim-nightly")]
luajit_bindings::with_state(|lstate| lstate),
```

Reply:
What's what I tried at first, but for some unknown reason, this makes `nvim_buf_get_text()` return *zero* lines instead. I have no idea how to go about debugging that, but anything but a null pointer here breaks the function. Same with `get_lines()`.

I should have mentioned that in the PR, but as a stop-gap measure the null pointer should be good enough until we figure out what's happening (it matches the behavior the function had before the changes in neovim).

Reply:
Sounds good. Can you mention this in both `Buffer::get_text()` and `Buffer::get_lines()` so that we know why we're using null pointers there?